amethyst_input/src/input_handler.rs

```diff
@@ -552,6 +554,37 @@ where
                     }
                 })
                 .unwrap_or(0.0),
+            Axis::Mouse {
+                axis,
+                over_extendable,
+                radius_x,
+                radius_y,
+            } => {
+                let current_pos = self.mouse_position.unwrap_or((0., 0.));
+                let last_pos = self.mouse_last_position.unwrap_or(current_pos);
+                let delta = match axis {
+                    // These calculations have to be inverses in order to point into the right direction of movement
+                    MouseAxis::X => last_pos.0 - current_pos.0,
+                    MouseAxis::Y => last_pos.1 - current_pos.1,
+                };
+
+                if over_extendable {
+                    return delta;
```

Comment:
I was actually thinking over extendable mouse axes would also be relative, they just wouldn't be clamped between 1 and -1. So after computing rel_delta, I would just return that if over_extendable. What are your thoughts on that approach?